local-storage/src/block.rs

```diff
@@ -5,7 +5,7 @@ use cid::Cid;
 use iroh_unixfs::Block;
 use std::str::FromStr;
 
-#[derive(Debug, PartialEq)]
+#[derive(Clone, Debug, PartialEq)]
```

Comment:
Expensive?

Reply:
I was being lazy but got this cleaned up.